roles/dnsmasq/templates/dnsmasq-autoscaler.yml.j2

```diff
@@ -54,3 +54,8 @@ spec:
             - --default-params={"linear":{"nodesPerReplica":{{ dnsmasq_nodes_per_replica }},"preventSinglePointFailure":true}}
             - --logtostderr=true
             - --v={{ kube_log_level }}
+{% if kube_patch_win_nodes %}
+      # When having win nodes in cluster without this patch, this pod cloud try to be created in windows
+      nodeSelector:
```

Comment:
isn't safe to always add this selector ?

Reply:
In the past I have sent merge request with the selector added without if and someone else asked to add something like this.

Reply:
Here I found @woopstar diliked the idea to have the nodeselector: https://github.com/kubernetes-incubator/kubespray/pull/2556

Reply:
where did you find this label: `beta.kubernetes.io/os` is it documented somewhere, is it used by other installer (kops, kubeadm ?)

Reply:
I think it's perfectly legit to add them, https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#beta-kubernetes-io-os